prysm/x/polarization.py

```diff
@@ -1,6 +1,19 @@
 "Jones and Mueller Calculus"
 from prysm.mathops import np
 from prysm.conf import config
+from prysm import propagation
+from prysm.coordinates import make_xy_grid,cart_to_polar
+import functools
+
+
+# supported functions for jones_decorator
+supported_propagation_funcs = ['focus','unfocus','focus_fixed_sampling','angular_spectrum']
+
+
+U = np.array([[1, 0,   0,  1],
```

Comment:
Please create this where it is needed (at most with a `def _U():; U = _U()`) -- by pre-computing it like this, GPU support is broken, since it will always reside on the device where "np" is at import time, which is always numpy-numpy.

Please also add `dtype=config.precision_complex` to the `array()` call

Reply:
Moved into `jones_to_mueller` and added the `dtype=config.precision_complex`